### DISCUSSION TITLE
any video player that have a lot of features...

### DISCUSSION THREAD

**Discussion by idontveapc** [nyas1/Material-You-app-list#75] [Q&A]
like, customisable and gesture features?

**Comment by ghost** [comment]
Hmm, I think your best bet would be [This mod of VLC](https://t.me/waifupx_official/233), (i dunno about gestures tho) unless you're including non-material design, Which this isn't the repository for.


Hope this helped ^⁠_⁠^

**Reply by idontveapc** [reply]
i knew the mod already, but is there mod for all format [__**xplayer**__](https://play.google.com/store/apps/details?id=video.player.videoplayer)? i has a _helpful_ _gesture_ `long press ff play` that i couldn’t find on other players, like mx.

_**anyway**_, **do you know any gboard or other material you based keyboard?** i’m already using _floris keyboard_ but im **tired** __editing themes__, :sad: do you know that comes with material you or easily customized??

**Reply by ghost** [reply]
> _**anyway**_, **do you know any gboard or other material you based keyboard?** i’m already using _floris keyboard_ but im **tired** __editing themes__, :sad: do you know that comes with material you or easily customized??

Sorry for being **really late**, lmao. But I've made a [material you theme of the Florisboard keyboard](https://github.com/TeaEndsAcronyms/Gboard-for-Florisboard-theme).

 [It requires this version of florisboard](https://www.mediafire.com/file/pywec1eo0iqze0j/dev.patrickgold.florisboard.debug.apk/file)

**Reply by fallsdevil** [reply]
> _**anyway**_, **do you know any gboard or other material you based keyboard?** i’m already using _floris keyboard_ but im **tired** **editing themes**, :sad: do you know that comes with material you or easily customized??

[heliboard](https://github.com/Helium314/HeliBoard) It's a great keyboard and has this Material You design and is customizable and is receiving updates
